src/app/app.component.css

```diff
@@ -0,0 +1,6 @@
+.layout {
+  margin: 30px auto 30px auto;
```

Comment:
When the values repeat you can use the shortcut `margin: 30px auto;`

Reply:
Thanks. I was trying different layouts with multiple values and forgot to change it.